prow/config.yaml

```diff
@@ -1,10 +1,7 @@
 ---
 plank:
-  # TODO(fejta): switch to spyglass
-  #ob_url_template: 'https://prow.istio.io/view/gcs/istio-prow/{{if eq .Spec.Type "presubmit"}}pull/{{.Spec.Refs.Org}}_{{.Spec.Refs.Repo}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}{{else}}.{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}/'
-  #ob_url_prefix: https://prow.istio.io/view/gcs
-  job_url_template: 'https://gubernator.k8s.io/build/istio-prow/{{if eq .Spec.Type "presubmit"}}pull/{{.Spec.Refs.Org}}_{{.Spec.Refs.Repo}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}{{else}}.{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}/'
-  job_url_prefix: https://gubernator.k8s.io/build/
+  job_url_template: 'https://prow.istio.io/view/gcs/istio-prow/{{if eq .Spec.Type "presubmit"}}pr-logs/pull/{{.Spec.Refs.Org}}_{{.Spec.Refs.Repo}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}{{else}}logs/{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}'
```

Comment:
We also need to handle batch results properly. It looks like they use `pr-logs/pull` instead of `logs/` and `/batch` instead of `/{{.Number}}`

Reply:
Done